README.md

```diff
@@ -10,7 +10,7 @@ extension providing tools and models for text analysis.
 
 The goal of this package is to provide an interface to various Natural Language Processing (NLP) resources for `MLJ` via such existing packages like [TextAnalysis](https://github.com/JuliaText/TextAnalysis.jl)
 
-Currently, we have  TF-IDF Transformer which converts a collection of raw documents into a TF-IDF matrix.
+Currently, we have a TF-IDF Transformer which converts a collection of raw documents into a TF-IDF matrix.  We also have a similar way of representing documents using the Okapi Best Match 25 algorithm - this works in a similar fashion to TF-IDF but introduces the probability that a term is relevant in a particular document.  See more here: [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25).  Finally, there is also a simple Bag-of-Word representation available.
```

Comment:
```suggestion
Currently, we have a TF-IDF Transformer which converts a collection of raw documents into a TF-IDF matrix. We also have a similar way of representing documents using the Okapi Best Match 25 algorithm - this works in a similar fashion to TF-IDF but introduces the probability that a term is relevant in a particular document.  See [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25). Finally, there is also a simple Bag-of-Word representation available.
```

Reply:
Yeah - I have typically done two spaces between sentences.  Not sure what the standard is if there is one.